frontend/src/context/login/LoginProvider.tsx

```diff
@@ -0,0 +1,24 @@
+import { ReactNode, createContext, useState } from 'react';
+
+import noop from '@utils/noop';
+
+interface LoginProviderProps {
+  children: ReactNode;
+}
+
+interface ContextType {
+  isLoggedIn: boolean;
+  setIsLoggedIn: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const hasAccessToken = !!window.localStorage.getItem('accessToken');
+
+export const LoginContext = createContext<ContextType>({
+  isLoggedIn: false,
+  setIsLoggedIn: noop,
+});
+
+export const LoginProvider = ({ children }: LoginProviderProps) => {
+  const [isLoggedIn, setIsLoggedIn] = useState(hasAccessToken);
```

Comment:
```suggestion
  const [isLoggedIn, setIsLoggedIn] = useState(() => !!window.localStorage.getItem('accessToken'));
```

이런건 어떨까요? 보기가 좋아서요 ~